app/controllers/journeys/nics/NICsCYAController.scala

```diff
@@ -49,11 +50,18 @@ class NICsCYAController @Inject() (override val messagesApi: MessagesApi,
     with I18nSupport
     with Logging {
 
+  // TODO uncomment onPageLoad lines when implementing the CYA content for multiple businesses
```

Comment:
Started doing all the CYA content, didn't realise the ticket was just for single business